var/spack/repos/builtin/packages/ocl-icd/package.py

```diff
@@ -42,6 +43,9 @@ class OclIcd(AutotoolsPackage):
     provides('opencl@:2.1', when='@2.2.8:2.2.11+headers')
     provides('opencl@:2.0', when='@2.2.3:2.2.7+headers')
 
+    patch('https://github.com/OCL-dev/ocl-icd/commit/aed1832c81c0971ea001e12d41e04df834257f94.patch',
```

Comment:
Can you add a comment above this patch directive to briefly explain what it does?

Reply:
done!